src/Utils.cpp

```diff
@@ -0,0 +1,19 @@
+#include <iostream>
+#include <string>
+#include <string.h>
+
+#include "Utils.h"
+
+using namespace std;
+
+
+string Utils::ConvertToString(char* a, int size) 
+{
+  int i; 
```

Comment:
you can(and should)  replace this whole function by using `std::string(a, size)` 
see https://en.cppreference.com/w/cpp/string/basic_string/basic_string (ctor number 4)